node/src/server-modules/GlideJson.ts

```diff
@@ -334,12 +334,12 @@ export class GlideJson {
      *
      * @example
      * ```typescript
-     * console.log(await GlideJson.set(client, "doc", "$", "[1, 2.3, "foo", true, null, {}, []]"));
+     * console.log(await GlideJson.set(client, "doc", "$", '[1, 2.3, "foo", true, null, {}, []]'));
      * // Output: 'OK' - Indicates successful setting of the value at path '$' in the key stored at `doc`.
      * const result = await GlideJson.type(client, "doc", "$[*]");
      * console.log(result);
      * // Output: ["integer", "number", "string", "boolean", null, "object", "array"];
-     * console.log(await GlideJson.set(client, "doc2", ".", "{Name: 'John', Age: 27}"));
+     * console.log(await GlideJson.set(client, "doc2", ".", '{Name: 'John', Age: 27}'));
```

Comment:
Nit: if you wrap the string with single quotes, you can use double quotes in the string without escaping, and vice versa. Now you're using single quotes inside a string wrapped by single quotes, that won't work